docker/docs/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM centos:7
+RUN yum install -y which gcc gcc-c++ kernel-devel make epel-release && \
+    yum install -y python36 && \
+    yum clean all
+ADD  https://bootstrap.pypa.io/get-pip.py /src/get-pip.py
```

Comment:
I think this might be a preferable way of getting pip (I *just* learned about it):

`python36 -m ensurepip`

afterwards, you should be able to refer to pip as `pip3`

Reply:
It seems like you might want to follow up with a `pip3 install -U pip` though

Reply:
I know the `get-pip` script fetches the latest version, but it's not a bad idea for `ensurepip` I guess. I'll look into how it works.

Reply:
It looks like we could do [`python36 -m ensurepip --upgrade`](https://docs.python.org/3/library/ensurepip.html#command-line-interface)

Reply:
I tried that, and it didn't do what I expected (I still ended up with pip 10)

Reply:
Interesting, same for me:

```
You are using pip version 10.0.1, however version 18.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
```

Alas.

Reply:
Using both `ensurepip` and `pip install -U pip` in da69ace!